features/cellular/framework/AT/AT_CellularStack.h

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2017, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef AT_CELLULAR_STACK_H_
+#define AT_CELLULAR_STACK_H_
+
+#include "AT_CellularBase.h"
+#include "NetworkStack.h"
+
+namespace mbed {
+
+// <PDP_addr_1> and <PDP_addr_2>: each is a string type that identifies the MT in the address space applicable to the PDP.
+// The string is given as dot-separated numeric (0-255) parameter of the form:
+// a1.a2.a3.a4 for IPv4 and
+// a1.a2.a3.a4.a5.a6.a7.a8.a9.a10.a11.a12.a13.a14.a15.a16 for IPv6.
+#define PDP_IPV6_SIZE 63+1
+
+/**
+ * Class AT_CellularStack.
+ *
+ * Implements NetworkStack and introduces interface for modem specific stack implementations.
```

Comment:
You should document that the current implementation supports only UDP here.

I see IPv6 code - is it functional?

Reply:
ticket created: IOTCELL-592